src/CatalogPage.js

```diff
@@ -1,10 +1,53 @@
+import { it } from "node:test";
+
 /**
  * This class obtains the elements in the DOM for the
  * catalog page using different locators
  */
 class CatalogPage {
   constructor(page) {
     this.productTitle = page.getByTestId("title");
+    this.productPrices = page.getByTestId("inventory-item-price");
+    this.dropdown = page.locator("//select[@data-test='product-sort-container']");
   }
+
+  async getPriceListArray(){
+    const priceItemsArray = [];
+    for(let item of await this.productPrices.all()){
+      const price  = await item.innerText();
+      const priceFloat = parseFloat(price.replace("$", ""));
+      priceItemsArray.push(priceFloat);
+    }
+    return priceItemsArray;
+  }
+
+  async orderArrayAscending(itemsPriceArray){
+    var pricceItemOrdered = itemsPriceArray.sort(function (a,b) {
+      return a - b; // Ascending
+  });;
+  return pricceItemOrdered;
+  }
+
+  async selectOrderBy(){
+    await this.dropdown.selectOption('Price (low to high)');
+  }
+
+  async compareArrays(pricesOrdered, pricesOrderByWebPage){
+      if(JSON.stringify(pricesOrdered) === JSON.stringify(pricesOrderByWebPage))
+        { return true;
+        }
+        else{
+          return false;
+        }
+  }
+
+  async valdateOrderByPriceLowToHigh(page){
```

Comment:
Typo: `valdate` -> `validate`

This is an interesting method we need to review in one of our sessions to provide you guidance.

Basically, this is your test case actually. It would be better to separate responsibilities in your code.

If you are planning to create validation methods, here is where the assertion would go.

I would suggest moving all the steps in this method to the test case and here is the assertion.

Or remove entirely this method and move the steps to the test case.